src/qtcore/qml/elements/QtQuick/MouseArea.js

```diff
@@ -81,6 +83,16 @@ registerQmlType({
         self.containsMouse = false;
         self.exited();
     }
+    this.dom.onmouseover = function(e) {
+        self.containsMouse = true;
+        self.dom.style.cursor = cursorShapeToCSS();
+        self.entered();
+    }
+    this.dom.onmouseout = function(e) {
+        self.containsMouse = false;
+        self.dom.style.cursor = "auto";
+        self.exited();
+    }
```

Comment:
Yes, I missed that to. That should have been an arrow function, transplited by babel =).
But lets use `self` for now.